GridMvc/Pagination/IGridPager.cs

```diff
@@ -24,6 +24,11 @@ public interface IGridPager
         /// </summary>
         int? ItemsCountOverwrite { get; set; }
 
+        /// <summary>
+        ///     Disables default paging if set to true
```

Comment:
What does it mean to disable default paging?

Reply:
Is now described better